Samples/HelloWorld.NetCore/src/OrleansClient/Program.cs

```diff
@@ -45,9 +48,11 @@ static int Main(string[] args)
             {
                 try
                 {
-                    var config = ClientConfiguration.LocalhostSilo();
+                    var siloAddress = IPAddress.Loopback;
+                    var gatewayPort = 30000;
                     client = new ClientBuilder()
-                        .UseConfiguration(config)
+                        .ConfigureCluster(options => options.ClusterId = "helloworldcluster")
+                        .UseStaticClustering(options => options.Gateways = new List<Uri>(){ (new IPEndPoint(siloAddress, gatewayPort)).ToGatewayUri() })
```

Comment:
NIT: 
`UseStaticClustering(options => options.Gateways.Add(new IPEndPoint(siloAddress, gatewayPort)).ToGatewayUri()) `

We are good people. We instantiate collections.